src/form/acp.jl

```diff
@@ -9,7 +9,19 @@ abstract type AbstractACPForm <: AbstractPowerFormulation end
 ""
 abstract type StandardACPForm <: AbstractACPForm end
 
-""
+"""
+AC power flow formulation with polar bus voltage variables.
+
+```
+@techreport{Cain2012,
+author = {Cain, Mary B and {O' Neill}, Richard P and Castillo, Anya},
+pages = {1--36},
+title = {{History of optimal power flow and formulations}},
+url = {https://www.ferc.gov/industries/electric/indus-act/market-planning/opf-papers/acopf-1-history-formulation-testing.pdf}
+year = {2012}
+}
+```
+"""
```

Comment:
Maybe we should also add,
```
@article{carpentier1962contribution,
  title={Contribution to the economic dispatch problem},
  author={Carpentier, J},
  journal={Bulletin de la Societe Francoise des Electriciens},
  volume={3},
  number={8},
  pages={431--447},
  year={1962}
}
```
it is the earliest refrence I know of.

Reply:
added